app/managers.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._conection = sqlite3.connect(r"D:\Games\cinema.db")
+
+    def all(self) -> list:
+        actors_data = self._conection.execute(
+            "SELECT * FROM actors"
+        )
+
+        return [row for row in actors_data]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._conection.execute(
+            "INSERT INTO actors (first_name, last_name) VALUES (?,?)",
+            (first_name, last_name)
+        )
+        self._conection.commit()
+
+    def update(self, id_: int, new_name: str, new_surname: str) -> None:
+        self._conection.execute(
+            "UPDATE actors "
+            "SET (first_name, last_name) = (?, ?) "
+            "WHERE id = ?",
+            (id_, new_name, new_surname)
```

Comment:
The parameter order in the `execute` method for the `update` function is incorrect. It should be `(new_name, new_surname, id_)` to match the SQL query's placeholders.